src/components/SecondaryPanes/CommandBar.css

```diff
@@ -4,6 +4,10 @@
   display: flex;
   height: 30px;
   overflow: hidden;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  background-color: white;
```

Comment:
oh right. Is that the var for the color? The naming scheme for them makes no sense so I have no clue what they are actually called.